src/packages/piral-base/src/dependency.ts

```diff
@@ -50,7 +50,7 @@ export function evalDependency(name: string, content: string, link = '', depende
 
   try {
     const sourceUrl = link && `\n//# sourceURL=${link}`;
-    const importer = new Function('module', 'exports', 'require', content + sourceUrl);
+    const importer = eval('(function anonymous(module,exports,require) {' + content + sourceUrl + '\n});\n');
```

Comment:
We've chosen `new Function` on purpose over `eval`. I see why you'd like to use it, but `eval` leaks context, while `new Function` does not.

Reply:
Understand too.